pkg/web/router.go

```diff
@@ -0,0 +1,41 @@
+package web
+
+import (
+	"github.com/aerogear/mobile-security-service/pkg/web/apps"
+	"github.com/labstack/echo"
+
+	"strings"
+
+	"github.com/labstack/echo/middleware"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+type RequestValidator struct {
+	validator *validator.Validate
+}
+
+func (v *RequestValidator) Validate(i interface{}) error {
+	return v.validator.Struct(i)
+}
+
+func NewRouter(fileDir string, apiRoutePrefix string) *echo.Echo {
+	router := echo.New()
+
+	router.Use(middleware.Logger())
```

Comment:
@damienomurchu The middleware is getting loaded twice, once here and in main.go. It should just occur here, using middleware.Init in my opinion.